docs/style.md

```diff
@@ -2,45 +2,53 @@
 
 This is a set of recommended InSpec rules you should use when writing controls.
 
-## Control files
+# Control Files
 
-### 1. All controls should be located in the "controls" directory and end in ".rb"
+## Place control files in "controls/" and end them with ".rb"
 
-Reason: Most syntax highlighters will render InSpec files correctly across a wide list of tools.
+Most syntax highlighters will render InSpec files correctly across a wide list
+of tools.
 
-Avoid: `controls/ssh_config`
-Use: `controls/ssh_config.rb`
+Avoid: `controls/ssh_config`</br>
+Use: `controls/ssh_config.rb`</br>
 
-Avoid: `controls/ssh/config.rb`
+Avoid: `controls/ssh/config.rb`</br>
 Use: `controls/ssh_config.rb`
 
-### 2. Avoid using "controls" or "control" in the name of your control files
+## Avoid "controls"/"control" in your control filenames
 
-Reason: Using `controls` in the filename again duplicates it and creates unnecessary clutter when reading it. Keep the names short and concise.
+Using `controls` in the filename creates unnecessary clutter when reading it.
+Keep the names short and concise.
 
-Avoid: `controls/ssh_controls.rb`
+Avoid: `controls/ssh_controls.rb`</br>
 Use: `controls/ssh.rb`
 
+# Code Style
 
-## Code style
-
-### 3. Avoid unnecessary parentheses in matchers
+## Avoid unnecessary parentheses in matchers
 
-Adding additional parentheses is not required and provides more readability if it is not used:
+Adding additional parentheses is not required and provides more readability if
+it is not used:
 
-Avoid: `it { should eq(value) }`
+Avoid: `it { should eq(value) }`</br>
 Use: `it { should eq value }`
 
 The exception are matchers that require additional arguments or named arguments.
 
+# Controls
 
-## Controls
+## Avoid wrapping controls in conditional statements
 
-### 4. Do not wrap controls in conditional statements
-
-Reason: This will create dynamic profiles whose controls depend on the execution. The problem here is that we cannot render the profile or provide its information before scanning a system. We want to be able to inform users of the contents of their profiles before they run them. It is valid to skip controls that are not necessary for a system, as long as you do it via `only_if` conditions. Ruby's internal conditionals will hide parts of the profile to static analysis and should thus be avoided.
+This will create dynamic profiles whose controls depend on the execution. The
```

Comment:
Why are we setting line length here (and other places in the file)? I think we should allow the renderer to wrap as it pleases. If it looks bad on render we should update those settings.

Reply:
It doesn't look bad on render either way but the file had both forced line wraps and no line wraps.

I went with forced line wraps for easier editing and making sure it was standard in the file...but I'm not set on that idea.

I prefer it because it makes reviewing in GitHub easier.

Reply:
`s/doesn't look bad either way/doesn't affect the render either way/`

It will render the same regardless of the line wraps in the file.